cfpb/cfpb.github.io
# Move Privacy Act statement to separate page

Also updates `Gemfile.lock` for the latest version of the GitHub Pages gem.

## Testing
1. Pull branch
2. `bundle install`
3. `bundle exec jekyll serve --baseurl ''`
4. Go to <http://localhost:4000/>
5. Click the "View Privacy Act statement" link near the bottom of the homepage to see the new page.

## Please review
- @cfarm 
- @marteki 
- @jimmynotjim
